netconfig/iwroute.py

```diff
@@ -4,13 +4,13 @@
 import asyncio
 from functools import partial
 
-from pr2modules.netlink.nl80211 import nl80211cmd
-from pr2modules.netlink.nl80211 import NL80211_NAMES
-from pr2modules.netlink import NLM_F_ACK
-from pr2modules.netlink import NLM_F_REQUEST
-from pyroute2.netlink.exceptions import NetlinkError
+from pr2modules.netlink.nl80211 import nl80211cmd  # noqa pylint: disable=no-name-in-module, import-error
```

Comment:
Does this have to be on each line or can we do something where we disable it at the beginning of the block and re-enable it at the end

For example (note example is with yapf not pylint):
https://github.com/ccxtechnologies/labrador/blob/206a7b5462bc8ab3dd36086ad7746c7c747cab7d/labrador_cacdf/m1553.py#L3
https://github.com/ccxtechnologies/labrador/blob/206a7b5462bc8ab3dd36086ad7746c7c747cab7d/labrador_cacdf/m1553.py#L1351